index.html

```diff
@@ -20,6 +20,16 @@
           <a><img class="menu-im" alt="Menu" width="24px" height="24px" src="images/menu.svg"/></a>
         </nav>
       </div>
+      <div class="overlay">
+        <div class="toolbar-2">
+          <img class="cancel1" alt="menu-cancel" width="24px" height="24px" src="images/cancel1.png">
+        </div>
+        <ul>
+          <li class="overlay-li poppins"><a class="a-li" href="#portfolio">Portfolio</a></li>
+          <li class="overlay-li poppins"><a class="a-li" href="#about-me">About Myself</a></li>
+          <li class="overlay-li poppins"><a class="a-li" href="#foot">Contact Me</a></li>
+        </ul>
+      </div>
```

Comment:
- [ ] It is required that, the design for your mobile menu must be exact like the Figma design. This includes  font, colors, images, text, margins, among others. Kindly ensure your design carries the same texts as we have them on the Figma design. 

### Your Design
![your-design](https://github.com/jcmunav63/mv_portfolio2/assets/26597432/6b0cf609-a74d-475c-986d-7c483f1075c8)

### Figma Design
![figma-design](https://github.com/jcmunav63/mv_portfolio2/assets/26597432/c16d77b8-819e-43d8-8502-e6795962f5c0)

- [ ] You will also need to reduce the left margin of the text so as to match the Figma design

### Your Design
![your-left-margin](https://github.com/jcmunav63/mv_portfolio2/assets/26597432/d42c9a1c-86b7-40e3-b191-075e52704dc8)

### Figma Design
![figma-left-margin](https://github.com/jcmunav63/mv_portfolio2/assets/26597432/e7e7eb95-8940-442e-ae77-17946b357a63)

- [OPTIONAL] It is the best user experience for the overlay on your mobile menu to be fixed. So that when the user scrolls up or down, the content under the overly only moves, not the overlay itself.

### App Behavior
[app-behave.webm](https://github.com/jcmunav63/mv_portfolio2/assets/26597432/76d66ef8-0ea1-4c9c-8d73-367eb0d9e21f)